### PR TITLE
Read python version from runtime.txt as well

### PR DESCRIPTION
**Pull request by Milo123459**
## What does this PR address?

Reads python version from runtime.txt

Fixes # (issue)

## Before submitting:

- [x] Make sure to follow [GitHub's guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on creating PR.
- [x] Did you read through [contribution guidelines](https://github.com/railwayapp/nixpacks/blob/main/CONTRIBUTING.md)?
- [ ] Did your changes require updates to the documentation? Have you updated those accordingly?
- [ ] Did you write tests to cover your changes? Did it pass locally when running `cargo test`?
- [ ] Have you run `cargo fmt`, `cargo check` and `cargo clippy` locally to ensure that CI passes?

### CODE REVIEW

**Review by JakeCooper — APPROVED** [review]

**Comment by Milo123459** [comment]
Not sure why Zig tests are failing, as nothing has changed for Zig. cc @coffee-cup?

**Review by coffee-cup — COMMENTED** [review]
Can you add a unit test for this too

**Comment by coffee-cup** [comment]
Looks like there was a problem fetching data from GitHub 
image

File: examples/python-2-runtime/runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7
```
Comment (coffee-cup) [inline examples/python-2-runtime/runtime.txt:1]: Can this have the format `python-2.7` to match the other runtime.txts

**Pull request merged**